components/motor/errors.go

```diff
@@ -30,3 +30,8 @@ func NewGoToUnsupportedError(motorName string) error {
 func NewControlParametersUnimplementedError() error {
 	return errors.New("control parameters must be configured to setup a motor with controls")
 }
+
+// NewSetRPMUnsupportedError returns an error when a motor is required to support SetRPM.
```

Comment:
comment seems incorrect?

Reply:
yeah I kind of agree but was trying to follow the pattern of the other comments/errors of this type

Reply:
I'll change it tho